src/client/client_impl.cc

```diff
@@ -19,19 +20,25 @@ void ClientImpl::cache_file_chunk_metadata(
       filename, chunk_index, chunk_handle));
   
   if (!set_chunk_handle_status.ok()) {
-    // TODO(Xi): if the above set fails, emit a log
+    LOG(WARNING) << "CacheManager failed to set chunk handle mapping for " 
```

Comment:
why not LOG(ERROR)?

Reply:
I guess it is a judgement call here. The client discards this and keeps going, but I have not problem to make it error